python/cudf/cudf/tests/test_copying.py

```diff
@@ -99,6 +99,14 @@ def test_series_setitem_cow(copy_on_write):
         assert_eq(actual, cudf.Series([1, 2, 300, 300, 5]))
     assert_eq(new_copy, cudf.Series([1, 2, 300, 300, 5]))
 
+    new_slice = actual[2:]
```

Comment:
Can we assert at this point that data are still shared, since that was the issue we were trying to fix here?

Reply:
Done 👍